charters/CommunicationsTeam/charter.md

```diff
@@ -0,0 +1,64 @@
+
+# Data Coordination Platform Communications Team
+
+
+## Description
+The Human Cell Atlas (HCA) Data Coordination Platform (DCP) Communications team coordinates and publishes public communications through various channels for the DCP as a whole. We work with the different groups within the DCP to ensure that each group is properly represented in external communications. We work with the HCA communications representatives if/when needed to ensure that DCP communications are consistent with and support those of the HCA. We monitor the DCP Data Portal documentation and update as needed. We develop training materials and events to teach people about the HCA DCP and how to use it.
+
+## Objective
+* Provide clear and accurate information to the public regarding DCP activities
+* Provide educational materials to help people to take advantage of DCP tools and HCA data
+* Function as the front line for communications between the DCP and legal or media entities, triaging requests or issues to the appropriate DCP or HCA leadership and possibly creating first drafts for their review (eg: press releases).
+* Support users and ensure that questions regarding DCP tools or HCA data are forwarded onto the appropriate group
+
+## In-scope
+* Update and maintain various public communication channels, for example:
+    * Social media channels
+    * Direct user communication
+    * Support applications (eg, Zendesk)
+* Develop and host educational materials  and events describing the DCP components
+* Monitor the state of our communications to ensure they are up-to-date and complete
+* Develop long-term plans for HCA DCP communication and estimate the resources needed to execute these plans
+
+### Scientific "guardrails"
+If we are communicating about any of the scientific activities of the HCA, the communication should be coordinated with the HCA project’s communications team.
+
+## Out-of-scope
+* Any communications for the HCA related to the scientific arm of the HCA project. 
+* UX research that informs us about the success of the portal or desired portal engineering 
+
+## Milestones
+* Announcement mailing list assembled
```

Comment:
Is there a strong rationale for why we would want an independent DCP mailing list? It would be convenient but I also worry that it could become confusing or fragmented.

Reply:
Great point - perhaps a subset does indeed make sense. Suppose my preference would be to try and generate this in parallel with the HCA email list (e.g. by someone opting in via the main registry) as opposed to forking and creating a new one.

Reply:
I think it makes a lot of sense to leverage a common onboarding for people to communication channels but I believe coordinated but separate communication efforts are important. We will be communicating different kinds of information at a different velocity than the HCA science arm. It is important people can opt in an out of the different communications and that our communications team is enabled to proceed as they need and do not have barriers when communicating DCP related material.

Reply:
I agree and I like Jonah's idea of generating the DCP email list via an option for users from the main registry. In addition to the ease for users, it gives us an opportunity to clarify to users what the DCP does as part of the HCA.